apps/webapp/app/routes/api.v1.$endpointSlug.schedules.$id.registrations.ts

```diff
@@ -3,7 +3,7 @@ import { json } from "@remix-run/server-runtime";
 import {
```

Comment:
These should be type imports or else moved outside of a route and into a .server file